### PR TITLE
[LDAP secret] Use secrets for LDAP main password

### PR DESCRIPTION
**Pull request by jeanpommier**
Support implemented by https://github.com/georchestra/georchestra/pull/3269

Pass LDAP admin password and geoserver_privileged_user's password using secrets (passwords should not be provided using plain env. variables.

### CODE REVIEW

File: secrets/gs_master_passwd.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+gerlsSnFd6SmM
```
Comment (fvanderbiest) [inline secrets/gs_master_passwd.txt]: I'd rename GS_MASTER into GEOSERVER_PRIVILEGED_USER to make it clear it's the same

Comment (fvanderbiest) [reply secrets/gs_master_passwd.txt]: Sorry, I should have seen that when reviewing https://github.com/georchestra/georchestra/pull/3269

**Review by fvanderbiest — CHANGES_REQUESTED** [review]
LGTM otherwise

**Comment by jeanpommier** [comment]
Seems reasonable. I've submitted a PR : https://github.com/georchestra/georchestra/pull/3413

I'll update this one accordingly once it has been merged

**Comment by fvanderbiest** [comment]
Excellent ! Thanks :-)

**Pull request merged**